libc/tzcode/localtime.c

```diff
@@ -274,7 +274,7 @@ settzname(void)
     tzname[1] = wildabbr;
 #ifdef USG_COMPAT
     daylight = 0;
-    timezone = 0;
+    timezone = -sp->ttis[sp->defaulttype].tt_gmtoff;
```

Comment:
Yes, it's true. 

The confusing nature of |timezone| is kept way up to the JS method [Date.getTimezoneOffset()](http://www.w3schools.com/jsref/jsref_gettimezoneoffset.asp). W3C gives this example:
"If your time zone is GMT+2, -120 will be returned."